specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/afdx.json

```diff
@@ -8214,7 +8208,7 @@
           "type": "object",
           "$ref": "#/definitions/DNSSettingsParameters"
         },
-        "domainHttps": {
+        "domainHttpsSettings": {
```

Comment:
Should we rename this to httpsSettings. The domain seems redundant.

Reply:
Sure. Will do.